test/CMakeLists.txt

```diff
@@ -15,6 +15,17 @@
 set (ranlux_24_48_test.pass_timeout_debug "900") # 15min
 set (ranlux_24_48_test.pass_timeout_release "720") # 12min
 
+include(CheckIncludeFileCXX)
+CHECK_INCLUDE_FILE_CXX("tbb/version.h" HAVE_TBB)
+find_program(FIND_GXX_EXE g++)
+
+if (FIND_GXX_EXE)
+    execute_process(COMMAND ${FIND_GXX_EXE} -dumpversion OUTPUT_VARIABLE _onedpl_gxx_version)
+endif()
+if (_onedpl_gxx_version EQUAL 11 AND HAVE_TBB AND NOT ONEDPL_BACKEND MATCHES "^(tbb)$")
```

Comment:
Please confirm you meant to make it exactly for g++ 11 and newer versions shouldn't be covered.

Reply:
After consulting, I came to the conclusion that it is more correct to cover version 11 and newer.
I added this to a commit

Reply:
It's been verified that the issue arises since g++ 9, so it is better to apply the workaround since its version.